### PR TITLE
Switch cook browse

### PR DESCRIPTION
**Pull request by ivg2**
Functionality between recipes that the user can make right away and recipes that can be made in the future.

### CODE REVIEW

File: app/src/main/java/me/ivg2/foodapp/RecipeFragment.java [hunk 1]
```diff
@@ -86,6 +95,59 @@ public void onClick(View view) {
...
+                        ArrayList<Recipe> tempRecipes = new ArrayList<>();
+                        for (int j = 0; j < 5; j++) {
+                            for (int i = 0; i < browsingRecipes.size(); i++) {
+                                if (browsingRecipes.get(i).getIngredientsMissing().size() == j) {
+                                    tempRecipes.add(browsingRecipes.get(i));
+                                }
+                            }
+                        }
+                        browsingRecipes = tempRecipes;
+                        //cookingRecipes = RecipeItemRepository.getAll();
```
Comment (bezi) [inline app/src/main/java/me/ivg2/foodapp/RecipeFragment.java]: dead code

File: app/src/main/java/me/ivg2/foodapp/RecipeFragment.java [hunk 2]
```diff
@@ -86,6 +95,59 @@ public void onClick(View view) {
...
+        tabs = view.findViewById(R.id.tab_host);
+        tabs.setSelectedTabIndicatorColor(getResources().getColor(R.color.colorAccent));
+        tabs.setTabTextColors(Color.parseColor("#000000"),getResources().getColor(R.color.colorAccent));
+        tabs.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
+            @Override
+            public void onTabSelected(TabLayout.Tab tab) {
+                refreshRecommendedRecipes();
+
+                switch (tab.getPosition()) {
+                    case 0:
```
Comment (bezi) [inline app/src/main/java/me/ivg2/foodapp/RecipeFragment.java]: not a big fan of case 0 and case 1: make a const called RECIPES_TO_COOK_POSITION = 0and RECIPES_CLOSE_TO_COOKING = 1, or something that is more clear.

File: app/src/main/java/me/ivg2/foodapp/RecipeFragment.java [hunk 3]
```diff
@@ -86,6 +95,59 @@ public void onClick(View view) {
...
+                        tab.getIcon().setColorFilter(Color.parseColor("#000000"), PorterDuff.Mode.SRC_IN);
+                        return;
+                    case 1:
+                        tab.getIcon().setColorFilter(Color.parseColor("#000000"), PorterDuff.Mode.SRC_IN);
+                        return;
+                }
+            }
+
+            @Override
+            public void onTabReselected(TabLayout.Tab tab) {
```
Comment (bezi) [inline app/src/main/java/me/ivg2/foodapp/RecipeFragment.java]: Do we need this funciton if it is empty?

Comment (ivg2) [reply app/src/main/java/me/ivg2/foodapp/RecipeFragment.java]: We need to implement the functions from the abstract class TabLayout.OnTabSelectedListener(). But I don't need anything in this one.

**Pull request merged**